client/src/components/_common/CustomMarkdown.tsx

```diff
@@ -0,0 +1,65 @@
+import { Box, Divider } from '@mui/material';
+import React, { FC } from 'react';
+import Markdown from 'react-markdown';
+import remarkGfm from 'remark-gfm';
+
+export interface CustomMarkdownProps {
+  description: string;
+  [key: string]: string; // other props
+}
+
+/*
+ * Markdown wrapper component to customize styling
+ */
+const CustomMarkdown: FC<CustomMarkdownProps> = ({ description, ...props }) => {
+  const descriptionLines = description.split('\n');
+  let lineIsQuote = false;
+  const quotes: string[] = [];
+  // Separate description into sections of blockquotes and non-blockquotes
+  descriptionLines.forEach((line) => {
```

Comment:
Wow this is a lot more complex than I thought.  Why can't this just be something like this (a very cursory search resulted in this, apologies if there is an obvious reason why this isn't supported...e.g. wrong library):

```jsx
     <Markdown 
        components={{
          blockquote: ({node, children}) => (
            <blockquote style={{borderLeft: "4px solid #ccc", paddingLeft: "1rem"}}>
              {children}
            </blockquote>
          )
        }}
      >
        {markdown}
      </Markdown>
```

Reply:
This is what I mean by the way:

![Screenshot 2024-11-26 at 1 07 51 PM](https://github.com/user-attachments/assets/f2adf8db-0721-4431-80ab-4baa0302a534)